eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -231,7 +231,7 @@ jobs:
             overWrite: true
 
     # Sign diagnostic files on Windows
-    - ${{ if and(eq(parameters.osGroup, 'windows'), ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), ne(parameters.testGroup, 'clrTools')) }}:
```

Comment:
I believe `clrTools` was needed here for something that David added. Can't recall if it was managed PGO or CrossGen2

Reply:
Also, if this is to fix the build perf pipeline (sorry, I was out) the same check should be done on line 135

Reply:
Last point was solved with your last commit.

Reply:
Clrtools is only used in the public build. We don't pass down clrtools as a test group in the official build

Reply:
I've made the equivalent fix on 135 as well.